hoodie-common/src/main/java/com/uber/hoodie/common/model/HoodieWriteStat.java

```diff
@@ -61,6 +61,11 @@
    */
   private long numUpdateWrites;
 
+  /**
+   * Total number of insert records converted to updates (for small file handling)
+   */
+  private long numInsertsAsUpdateWrites;
```

Comment:
thinking on the naming here, I feel we should just leave it as `numInserts` and also populate it from CreateHandle to be consistent.. We can then use things like prevCommit etc to tell whether a WriteStat is from Merge/Append/Create correct? I feel this is better than leaking implementation details like smal file handling into the metadata.. `numUpdateWrites`  sets a precedent already